lib/flip_card.dart

```diff
@@ -51,6 +51,8 @@ typedef void BoolCallback(bool isFront);
 class FlipCard extends StatefulWidget {
   final Widget front;
   final Widget back;
+  //to add same decorations for each sides of card
```

Comment:
hi, thx for your reply. Builder is not alternative for front and back widgets. It is useful way to wrap each widget in the same decorations(widgets like DecoratedBox, ColoredBox and others). For example: 
dart```
final w1 = Widget1();
final w2 = Widget2();
return FlipCard(
  front: w1,
  back: w2,
  builder: (child, isFront){
  /// is Front is not required to use
  return DecoratedBox(
    decoration: BoxDecoration....,
    child: child, // where child is front and back wrapping in same color, border radius and other things...
  )
}

Reply:
/// added for explanation of the purpose for builder

Reply:
Wouldnt it make more sense to use builder as a replacement for front and back? If it's "just" for decorations but could be used to completely replace them, then why allow both at the same time?

Reply:
What I mean is:
Why pass around the builder child when the package user can just define the child inside the builder callback in a more flexible way

Reply:
I think is allowing user to define what to return from builder using if statement based on isFront flag is more complicated than just to wrap current child(front or back). Also front/back state already defined in current flipCard widget. If you want example of this way of composing, please,  look at ValueListenableBuilder. Maybe renaming child to currentChild will be more obvious for user?

Reply:
If statements in builder looks not very good and bad to read, in my opinion

Reply:
Well, if you want I can change merge request to make builder more flexible

Reply:
- I think currentChild is better
- I still think extra flexibility is good here. Automatically using front/back has no real benefit other than saving yourself from creating 2 variables for front and back page if you haven't already done that
- Keep in mind I'm not maintaining this, I'm just a collaborator giving his opinion on the PR

Reply:
ok

Reply:
Well, I decided to use named constructors with required parameters to avoid assertions. Btw, in the future make a factory will be better but now for this small feature